css/main.css

```diff
@@ -395,16 +395,839 @@ SOFTWARE.
+html {
```

Comment:
Could you revert this. I'd like to keep the copy-pasted CSS unchanged.

Reply:
Sorry about that. Prettier is enable on file save, hence the changes across.